src/ofApp.cpp

```diff
@@ -90,7 +92,7 @@ void ofApp::keyReleased(int key){
 
 //--------------------------------------------------------------
 void ofApp::mouseMoved(int x, int y ){
-    frameWidth = abs(320 - x) + 1;
```

Comment:
should this be `abs(camWidth / 2 - x) + 1`?

Reply:
Don't know. If divided by two you can never scan the full web cam width.

Reply:
right! I think this:
`abs(camWidth / 2 - x) * 2 + 1` will do what I intended -- distance from centre, but follow mouse to the full width on both sides. But as you said, it's totally whatever UX you like.